src/js/components/pages.js

```diff
@@ -348,3 +352,53 @@ export class Service {
 }
 
 export class ServicePointPage extends Service {}
+
+export class NoticeIndex {
+  constructor(content) {
+    this.breadcrumbItems = getBreadcrumbsWithLabel(content.ancestor_pages);
+    this.childPages = content.child_pages;
+  }
+
+  renderChildPageLinks() {
+    const childPageLinks = this.childPages.map((page) => {
+      const props = {
+        title: page.title,
+        url: page.url,
+        subtitle: "",
+      };
+
+      return new LinkBlock(props);
+    });
+
+    if (childPageLinks.length) {
+      return [new FullWidthGrid(childPageLinks).render()];
+    } else {
+      return [];
+    }
+  }
+
+  render() {
+    return [
+      new Breadcrumbs(this.breadcrumbItems).render(),
+      ...this.renderChildPageLinks(),
+    ];
+  }
+}
+
+export class NoticePage {
+  constructor(content) {
+    this.breadcrumbItems = getBreadcrumbsWithLabel(content.ancestor_pages);
+    this.noticeMainContent = content.body_html;
+    this.publicationDate = content.publication_date;
+  }
+
+  render() {
+    return [
+      new Breadcrumbs(this.breadcrumbItems).render(),
+      new SectionHeading("Overview").render(),
+      new ExpandableRichText(this.noticeMainContent).render(),
+      new SectionHeading("Publication Date").render(),
+      timeElem(this.publicationDate),
```

Comment:
Instead of a heading, can we put the date at the top inside the rich text block without a heading as in the mockup?

Reply:
I have not seen the mock-up :) will have a look. Is it inside the Figma document?

Reply:
Sorry, I've been blasting you with slack messages and hoping you'd update the issues and trello as needed

Reply:
https://openupsa.slack.com/archives/C0189R440L9/p1607685421033900

https://openupsa.slack.com/archives/C0189R440L9/p1607685450034300